ui/src/app/views/cohort-edit/component.spec.ts

```diff
@@ -1,25 +1,21 @@
 import {Component, DebugElement} from '@angular/core';
-import {TestBed, async, fakeAsync, tick, ComponentFixture} from '@angular/core/testing';
+import {TestBed, async, fakeAsync, ComponentFixture} from '@angular/core/testing';
 import {Title, By} from '@angular/platform-browser';
 import {FormsModule} from '@angular/forms';
 import {ActivatedRoute, UrlSegment} from '@angular/router';
 import {CohortEditComponent} from 'app/views/cohort-edit/component';
 import {RouterTestingModule} from '@angular/router/testing';
 import {BehaviorSubject} from 'rxjs/BehaviorSubject';
-import {SignInService} from 'app/services/sign-in.service';
 import {WorkspaceComponent} from 'app/views/workspace/component';
-
-import {Cohort, CohortListResponse} from 'generated';
+<<<<<<< HEAD
```

Comment:
Oh! For some reason my text editor wasn't showing these, but it could just be it wasn't updating. 
I'm surprised the linter didn't catch it and prevent the push.